src/api.rs

```diff
@@ -123,6 +130,10 @@ pub fn process_from_json_string(input: &str) {
                         output: FileInfo {
                             path: processing_info.path,
                         }
+                    },
+                    metadata: MetadataPayload {
+                        // TODO: This unwrap is theoretically safe, but it would be nice to have it checked by borrow checker, or handled in some other way
+                        date: datetime_to_iso_string(&processing_info.exif_data.unwrap().datetime),
```

Comment:
Co znaczy date? CreatedDate, modifiedDate? Ja bym uszczegółowił

Dodatkowo nazywasz pole Date a trzymasz w nim datetime ;)

To może zwykły createdAt

Reply:
W tym kontekście zdjęcia chodzi o datę jego wykonania, jedyną datę jaka jest interesująca kiedy mówimy o zdjęciu. Dlatego po prostu `date`.

A Datetime to smutna nazwa structa, którego mam z zewnętrznego crate, więc tu trochę nie mam wpływu. Wybrałem `date`, bo taka jest nazwa klasy w JS i zrównałem do tego.